src/components/AddProduct.js

```diff
@@ -0,0 +1,28 @@
+import ProductForm from "./ProductForm";

```

Comment:
Do you prefer the import like this or should we use the re-export like ...

import { ProductForm } from "./";